pkg/runtime/runtime.go

```diff
@@ -287,24 +289,27 @@ func newDaprRuntime(ctx context.Context,
 		},
 	)
 
-	var schedulerManager *runtimeScheduler.Manager
 	if runtimeConfig.SchedulerEnabled() {
-		if err = channels.Refresh(); err != nil {
```

Comment:
I ran into issues where the app channel was not initialized at this point. Did I miss you initializing it somewhere else?